src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -28,21 +28,20 @@ const EditShiftHours = ({ setShowEditShiftHours }) => {
   const onSubmit = async (formData) => {
     dayjs.extend(utc);
 
-    const actualStartDate = dayjs(formData['startDate']).format('YYYY-MM-DD');
+    const actualStartDate = dayjs(timecardDate).format('YYYY-MM-DD');
     const startTime = formData[`${inputName}-start-time`];
     const actualStartDateTime = dayjs(
       actualStartDate + ' ' + startTime
     ).format();
 
     const endTime = formData[`${inputName}-finish-time`] || null;
-    let actualEndDateTime;
-    if (endTime) {
-      actualEndDateTime = dayjs(actualStartDate + ' ' + endTime).format();
-    }
+    let actualEndDateTime = endTime
```

Comment:
Nice work changing this to avoid having to reassign the variable on a separate line 👍 
```suggestion
    const actualEndDateTime = endTime
```